src/scripts/main.js

```diff
@@ -1,3 +1,50 @@
 'use strict';
 
-// write code here
+const listOfElements = document.getElementsByTagName('li');
```

Comment:
The `listOfElements` is a live HTMLCollection, not an array. This could lead to unexpected behavior when sorting, as the HTMLCollection is updated automatically when the DOM changes. Consider converting it to an array using `Array.from()` before passing it to the `sortList` function.